client/src/components/FeedbackItem/FeedbackItem.tsx

```diff
@@ -1,22 +1,24 @@
 import React, { FC } from 'react';
 import { Feedback } from 'types';
-import { BsStar, BsStarFill } from 'react-icons/bs';
+import InputRating from 'components/InputRating/InputRating';
 
 type ItemProps = {
   item: Feedback;
 };
 
 const FeedbackItem: FC<ItemProps> = ({ item }) => {
-  const starArray = [...Array(5).keys()].map(i => {
-    return i + 1;
-  });
-
   return (
     <div>
-      {starArray.map(i => {
-        return item.rating >= i ? <BsStarFill key={i} /> : <BsStar key={i} />;
-      })}
-      <h3>{item.title}</h3>
+      <InputRating
+        type="hidden"
+        name="postRating"
+        id="postNum"
+        value={item.rating}
+        onChange={() => {
```

Comment:
you can mark onChang prop as optional

Reply:
Yes, I have done that, but then I got this error in the console
![Screenshot 2022-09-23 131435](https://user-images.githubusercontent.com/55662628/191939963-6b3cb2b1-f01b-4fe2-8537-7afc97874cb3.png)

Reply:
No, it is the same result with disabled. But I removed checked.